gadget/gadget.go

```diff
@@ -45,10 +45,11 @@ const (
 	// GPT identifies a GUID Partition Table partitioning schema
 	GPT = "gpt"
 
-	SystemBoot = "system-boot"
-	SystemData = "system-data"
-	BootImage  = "bootimg"
-	BootSelect = "bootselect"
+	SystemBoot     = "system-boot"
+	SystemData     = "system-data"
+	SystemRecovery = "system-recovery" // FIXME: this role name may be changed
```

Comment:
This will be SystemSeed and "system-seed"  as discussed on the paris sprint)

Reply:
Role name changed.